client/src/AppBundle/Resources/views/Report/Report/_header.html.twig

```diff
@@ -1,18 +1,30 @@
 {% set translationDomain = "report-overview" %}
 {% trans_default_domain translationDomain %}
 
+
 {% if reportStatus.isReadyToSubmit and report.isDue %}
-    <div id="header-report-submit" class="govuk-panel govuk-panel--confirmation behat-region-report-ready-banner">
-        {% if report.unSubmitDate and report.submitted == false %}
+
+    {% if report.unSubmitDate and report.submitted == false %}
+        <div id="header-report-submit" class="govuk-panel govuk-panel--confirmation behat-region-report-ready-banner">
             <h2 class="govuk-tag opg-tag--warning govuk-!-margin-bottom-6">{{ 'changesNeeded' | trans({}, 'common') }}</h2>
-        {% else %}
-            <h2 class="govuk-panel__title">{{ 'reportIsReady' | trans() }}</h2>
-        {% endif %}
-        <div class="govuk-panel__body">
-            <a class="govuk-button govuk-!-margin-0 behat-link-report-submit" href="{{ path('report_review', {'reportId': report.id}) }}" role="button" data-module="govuk-button">
-                {{ 'continue' | trans({}, 'common') }}
-            </a>
+        </div>
+    {% endif %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="moj-banner moj-banner--success">
+                <div class="moj-banner__message">
+                    <span class="moj-banner__assistive">Success</span>
+                    {{ 'reportIsReady' | trans() }}
+                </div>
+            </div>
         </div>
     </div>
```

Comment:
Should this be in `{% else %}`? Otherwise if the report is unsubmitted then users will see both the "changes needed" and "ready to submit" boxes immediately